onboarding-modal-experiment.toml

```diff
@@ -0,0 +1,97 @@
+[experiment]
+
+segments = ["attributed", "non_attributed"]
+
+enrollment_period = 14
```

Comment:
How was this 14 picked? The enrollment for this experiment was only 7 days as scheduled
https://experimenter.services.mozilla.com/nimbus/onboarding-modal-experiment

Maybe accidentally copied from some other onboarding experiment or intended for something else?